resources/language/resource.language.pt_br/strings.po

```diff
@@ -2479,278 +2479,303 @@ msgctxt "#32484"
 msgid "TVDb API"
 msgstr "TVDb API"
 
+#: /resources/tmdbhelper/lib/script/sync.py
 msgctxt "#30030"
-msgid "Hindi (India)"
+msgid "Add rating"
 msgstr "Hindi (Índia)"
 
+#: /resources/tmdbhelper/lib/api/tmdb/discover.py
 msgctxt "#30031"
-msgid "Hungarian (Hungary)"
+msgid "Certification"
 msgstr "Húngaro (Hungria)"
 
+#: /resources/tmdbhelper/lib/api/tmdb/discover.py
 msgctxt "#30032"
-msgid "Indonesian (Indonesia)"
+msgid "Certification ( < or = )"
 msgstr "Indonésio (indonésio)"
```

Comment:
It looks like something has gone wrong with your msgid values...

Reply:
> It looks like something has gone wrong with your msgid values...

This time I tried a software to facilitate the translation, I'm sorry, I'll try to correct it.